docs/pages/database-access/guides/mysql-self-hosted.mdx

```diff
@@ -176,8 +176,11 @@ a single self-hosted MySQL database:
 
 ```yaml
 teleport:
-  data_dir: /var/lib/teleport-db
+  data_dir: /var/lib/teleport
   nodename: test
+  # Teleport invitation token used to join a cluster.
+  # can also be passed on start using --token flag
+  auth_token: xxxx-token-xxxx
```

Comment:
The token is passed on the CLI below, we don't want to promote hardcoding tokens in the config file among users. Do you know if config file supports loading tokens from files?

Reply:
it does, if you provide a path: `/path/to/token`

Reply:
I like the intent to move away from hardcoded tokens. It would be good to review back @awly proposal to make obtaining machine readable output easier with something like `tctl db add --format=json -o` https://github.com/gravitational/teleport/issues/3732#issuecomment-631139040

For now, I'll add a note on how to load the invite token using a file.